cmd/evm/internal/t8ntool/execution.go

```diff
@@ -448,6 +450,11 @@ func MakePreState(db ethdb.Database, chainConfig *params.ChainConfig, pre *Prest
 		if err != nil {
 			panic(err)
 		}
+
+		if _, ok := statedb.GetTrie().(*trie.VerkleTrie); ok {
+			return statedb
+		}
+
```

Comment:
same thing here: add a command that this means the transition has completed and that we don't know to activate all the boilerplate that is necessary for it.